src/platform/datapath_epoll.c

```diff
@@ -1816,6 +1864,11 @@ CxPlatSocketContextRecvComplete(
                     CXPLAT_DBG_ASSERT_CMSG(CMsg, uint8_t);
                     TOS = *(uint8_t*)CMSG_DATA(CMsg);
                     FoundTOS = TRUE;
+                } else if (CMsg->cmsg_type == IP_TTL) {
+                    CXPLAT_DBG_ASSERT_CMSG(CMsg, uint8_t);
```

Comment:
The length of the IP_TTL ancillary data payload seems sizeof(int) as you use `CMSG_SPACE(sizeof(int))` to reserve the control buffer space.

Why are we not asserting it's a uint8?

Reply:
good catch! I wasn't sure reserving X space maps to getting X sized payload back. I will remove this assert and keep the cast to uint8_t